lib/ript/dsl/primitives/filter.rb

```diff
@@ -80,7 +79,6 @@ def build_rule(name, block, opts={})
               to_address    = @labels[to][:address]
 
               attributes = {
-                             "table"       => "filter",
```

Comment:
What's the reasoning behind omitting the table attribute?

The original reason it was included was to be explicit about what rules are being generated, and thus future proof Ript. If the future behaviour of iptables changes, and the filter table is no longer the default table, then Ript will silently break. 

Bonus: if you add this attribute back back, your PR will be way smaller because you won't have to modify all the tests.